fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/FDBRecordStoreBase.java

```diff
@@ -621,12 +622,10 @@ default boolean recordExists(@Nonnull final Tuple primaryKey, @Nonnull final Iso
      * @return a cursor that will scan the index, picking up at continuation, and honoring the given scan properties
      */
     @Nonnull
-    default RecordCursor<IndexEntry> scanIndex(@Nonnull Index index, @Nonnull IndexScanType scanType,
+    RecordCursor<IndexEntry> scanIndex(@Nonnull Index index, @Nonnull IndexScanType scanType,
                                                @Nonnull TupleRange range,
                                                @Nullable byte[] continuation,
-                                               @Nonnull ScanProperties scanProperties) {
-        return scanIndex(index, scanType, range, continuation, scanProperties, null);
-    }
+                                               @Nonnull ScanProperties scanProperties);
```

Comment:
These parameters should align with the `index` parameter on the first line of this function definition.